viewer/vue-client/src/components/mixins/facet-mixin.vue

```diff
@@ -0,0 +1,85 @@
+<script>
+import { mapGetters } from 'vuex';
+import * as MathUtil from '@/utils/math';
+import * as VocabUtil from '@/utils/vocab';
+import LensMixin from '@/components/mixins/lens-mixin';
+
+export default {
+  mixins: [LensMixin],
+  props: {
+  },
+  data() {
+    return {
+    };
+  },
+  methods: {
+    getByLang(object, property, lang) {
+      const langDict = object[`${property}ByLang`];
+      if (typeof langDict === 'object' && typeof langDict[lang] === 'string') {
+        return langDict[lang];
+      }
+      return object[property];
+    },
+    getRecordType(object) {
+      return VocabUtil.getRecordType(
+        object['@type'],
+        this.resources.vocab,
+        this.resources.context,
+      );
+    },
+    determineLabel(object) {
+      if (object.hasOwnProperty('mainEntity')) {
+        object = object.mainEntity;
+      }
+      const lang = this.user.settings.language;
+      
+      for (const prop of ['@id', '_key']) {
+        if (object.hasOwnProperty(prop)) {
+          const chains = this.settings.propertyChains;
+          const id = object[prop];
+          if (chains.hasOwnProperty(id)) {
+            return chains[id][this.user.settings.language];
+          }
+        }
+      }
+
+      if (object.hasOwnProperty('propertyChainAxiom')) {
+        return object.propertyChainAxiom
+          .map(o => this.$options.filters.capitalize(this.determineLabel(o)))
+          .join('/');
```

Comment:
How do we want to present this?
Now:
"Metadata/Katalogiseringsregler (446)"
"Seriemedlemskap/Ingår i serie/Instans av Verk (16k)"

Reply:
Can't think of anything better at the moment. Maybe best answered by users during testing? So I say we keep it like this for now.